setup.py

```diff
@@ -30,6 +32,7 @@
         'ckanapi.tests.mock',
         'ckanapi.cli',
         ],
+    setup_requires='requests',
```

Comment:
Without `setup_requires='requests'` setup will mistakenly identify `requests-toolbelt` as a match for `requests` and fail to install the latter. This appears to be a bug in setuptools. Diagnosis and workaround from [stackoverflow](http://stackoverflow.com/questions/27497470/setuptools-finds-wrong-package-during-install#answer-27582199).